Description: some current fixes from upstream
 * mouse marks and last line behaviour
 * font size 10 instead 12
Author: Alf Gaida <agaida@siduction.org>

--- qtermwidget-0.6.0.orig/src/main.cpp
+++ qtermwidget-0.6.0/src/main.cpp
@@ -48,7 +48,7 @@ int main(int argc, char *argv[])
 #else
     font.setFamily("Monospace");
 #endif
-    font.setPointSize(12);
+    font.setPointSize(10);
 
     console->setTerminalFont(font);
 
